Rear Rider/Rear Rider/ConfigurationOptions/OptionsView.swift

```diff
@@ -0,0 +1,131 @@
+            set: { newValue in
+                self.wrappedValue = newValue
+                handler(newValue)
+            }
+        )
+    }
+}
+
+struct OptionsView: View {
+    @ObservedObject var conf = UserConfig()
+    @State var confAudio: String = ""
+    @State var confLightPattern: String = ""
+    @State var confLightBrightness: Int = 1
+    @State var confLightColor: Color = .white
+    
+    let audioFiles: [ConfigOptions.AudioFile] = ConfigOptions.AudioFile.allCases
+    let lightPatterns: [ConfigOptions.LightPattern] = ConfigOptions.LightPattern.allCases
+    let lightBrightness: [ConfigOptions.LightBrightness] = ConfigOptions.LightBrightness.allCases
+    
+    let alert = RearRiderAlerts()
+    
+    // need to wrap these in this init to get access to self for the conf variable and set these
+    init() {
+        self._confAudio = State(wrappedValue: conf.audioFile)
+        self._confLightPattern = State(wrappedValue: conf.lightPattern)
+        self._confLightBrightness = State(wrappedValue: conf.lightBrightness)
+        self._confLightColor = State(wrappedValue: Color.fromRGBString(rgbString: conf.lightColor))
+    }
+    
+    var body: some View {
+        VStack {
+            Text("Options")
+            List {
+                Section {
+                    Picker("Audio Alert", selection: $confAudio.onChange(setAudio), content: {
+                        ForEach(audioFiles, id: \.self) {
+                            audioFile in Text(audioFile.description).tag(audioFile.rawValue)
+                        }
+                    })
+                } header: {
+                    Text("Audio")
+                }
+                Section {
+                    Picker("Pattern", selection: $confLightPattern.onChange(setLights), content: {
+                        ForEach(lightPatterns, id: \.self) {
+                            lightPattern in Text(lightPattern.description).tag(lightPattern.rawValue)
+                        }
+                    })
+                    Picker("Brightness", selection: $confLightBrightness.onChange(setBrightness), content: {
+                        ForEach(lightBrightness, id: \.self) {
+                            brightnessLevel in Text(brightnessLevel.description).tag(brightnessLevel.rawValue)
+                        }
+                    })
+                    ColorPicker("Color", selection: $confLightColor.onChange(setColor), supportsOpacity: false
+                    )
+                } header: {
+                    Text("Lights")
+                }
+            }
+        }
+    }
+    
+    // these are written out separately as to not clog up the view itself
+    /**
+     * Sets the config object's audio to the new selection and saves it
+     */
+    func setAudio(to value: String) {
+        conf.audioFile = confAudio
+        saveConf()
+        // play audio sound
+        if !conf.audioFile.isEmpty {
+            try! alert.loadSoundFile(fileName: conf.audioFile)
```

Comment:
If RearRiderAlerts gets moved to Rear_RiderApp then I think we need an init() method to pass the sound file loaded from the configuration so it's ready to be played at anytime.